var/spack/repos/builtin/packages/geos/package.py

```diff
@@ -40,17 +43,15 @@ class Geos(AutotoolsPackage):
     variant('python', default=False, description='Enable Python support')
 
     extends('ruby', when='+ruby')
+    extends('python', when='+python')
 
     # Python 3 is supposedly supported, but I couldn't get it to work
     # https://trac.osgeo.org/geos/ticket/774
```

Comment:
@citibeth It looks like this bug has been fixed in 3.5.2? I tried building with Python 3 and configure crashed because it couldn't find the library name, but technically I think it should support Python 3 now. Just one more package off the list of packages that only supported Python 2.